016_IP_L151/016_IP_L151_safe.py

```
@@ -0,0 +1,366 @@
+                    if(description[i][1]==l):
+                        edit(l,description[i][0],new_id,name)
+                    else:
+                        edit(description[i][1],description[i][0])
+        return render_template(f'admin_science.html',name=name,description=dec,l=l)
+    else:
+        return "您没有权限"
+@app.route('/user')
+def user_net():
+    cookie = request.cookies.get("username")
+    cookie2 = request.cookies.get("ss")
+    if cookie==None:
```

Ruff (0.14.1)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)